server/views/pages/staffDetail.njk

```diff
@@ -31,9 +31,9 @@
                     <td class="govuk-table__cell">{{ staffDetail.staffCode }}</td>
                     <td class="govuk-table__cell">{{ staffDetail.staff.forenames }}</td>
                     <td class="govuk-table__cell">{{ staffDetail.staff.surname }}</td>
-                    <td class="govuk-table__cell">{{ staffDetail.email }}</td>
+                    <td class="govuk-table__cell">{{ staffDetail.email or "NA" }}</td>
```

Comment:
The NA here isn't pretty - is it a common MOJ pattern?

Reply:
Not common, mostly is "Not entered" or left blank, but as its just a spike ticket I guess its ok for now?